cmd/semTable.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+    "fmt"
+    "github.com/anaskhan96/soup"
+)
+
+// semTable function - equivalent to your Python code
+func semTable(url string) {
+    fmt.Println("Fetching semesters...")
+    resp := fetchHTML(url)  // Fetch the HTML content
```

Comment:
Comment not needed. The function name is self explanatory.

Reply:
I meant for `fetchHTML`

Reply:
This code snippet has been updated.